doc/old/private_data_elements.rst

```diff
@@ -113,6 +113,18 @@ with block object::
     >>> 0x01 in block
     False
 
+Since pydicom 3.0, there is also a convenience method to add a private tag without the need
+of creating a private block first::
+
```

Comment:
Could simplify wording a little: '...a convenience method to add a private tag without creating a private block first'